.github/workflows/teardown_pull_request.yml

```diff
@@ -8,28 +8,32 @@ jobs:
   teardown:
     runs-on: ubuntu-20.04
     env:
-      IMAGE_NAME: ghcr.io/${{ github.repository_owner }}/scaleway-ui
-      REMOTE_SSH_SERVER: ssh://root@react.ui.scaleway.com
+      IMAGE_NAME: rg.fr-par.scw.cloud/consolescw/scaleway-ui
+      SCW_DNS: react.ui.scaleway.com
+      SCW_NAMESPACE_ID: f73b51d0-5a1d-42db-aad7-5c4ff86cb210
+
     steps:
       - uses: actions/checkout@v2.4.0
+
       - name: Inject slug/short variables
         uses: rlespinasse/github-slug-action@v4
+
       - name: Export custom variables
         run: |
           SAFE_GITHUB_HEAD_REF_SLUG_URL=$(echo $GITHUB_HEAD_REF_SLUG_URL | rev | cut -c-37 | rev)
           ([[ $GITHUB_REF == 'refs/heads/main' ]] && echo "BRANCH_SLUG=main" || echo "BRANCH_SLUG=$SAFE_GITHUB_HEAD_REF_SLUG_URL") >> $GITHUB_ENV
 
-      - name: Set Docker context
-        uses: arwynfr/actions-docker-context@v2
+      - name: Delete Serverless Container Scaleway
+        uses: philibea/scaleway-containers-deploy@v1.0.4
+        id: teardown
         with:
-          docker_host: ${{ env.REMOTE_SSH_SERVER }}
-          context_name: v-${{ env.BRANCH_SLUG }}
-          ssh_cert: ${{ secrets.SSH_CERT }}
-          ssh_key: ${{ secrets.SSH_KEY }}
-      - name: Remove deployment
-        env:
-          VERSION: v-${{ env.BRANCH_SLUG }}
-        run: docker-compose -p ${VERSION} --context ${VERSION} down --rmi all
```

Comment:
This means we can remove the docker-compose :)

Reply:
Yes, right, i just need to handle an other case when we want to deploy on our main dns actually it's not what we want.